Library/Homebrew/test/os/mac/java_requirement_test.rb

```diff
@@ -0,0 +1,29 @@
+require "testing_env"
+require "requirements/java_requirement"
+
+class OSMacJavaRequirementTests < Homebrew::TestCase
```

Comment:
Opinion question: would it make more sense to extend from the integration test class?